projects/plugins/social/src/class-note.php

```diff
@@ -104,4 +105,26 @@ public static function may_be_flush_rewrite_rules( $force = false ) {
 	public static function delete_rewrite_rules_option() {
 		delete_option( self::JETPACK_SOCIAL_REWRITE_RULES_LAST_FLUSHED_AT );
 	}
+
+	/**
+	 * Use the_title hook so we show the social note's exceprt in the post list view.
+	 *
+	 * @param array $title The title of the post, which we have set to be an empty string for Social Notes.
+	 * @param array $post_id The Post ID.
+	 */
+	public static function override_empty_title( $title, $post_id ) {
+		if ( is_admin() && 'edit.php' === $GLOBALS['pagenow'] ) {
```

Comment:
I'm not sure if it's better, but we can get the current screen with [`get_current_screen` ](https://developer.wordpress.org/reference/functions/get_current_screen/)and look at [the `base` property to check if it's `edit`](https://developer.wordpress.org/reference/classes/wp_screen/)